lib/libfrr.c

```diff
@@ -693,6 +693,9 @@ void frr_run(struct thread_master *master)
 		daemon_ctl_sock = -1;
 	}
 
+	/* end fixed stderr startup logging */
```

Comment:
I specifically think we don't want to; I have several reasons for stance:

- we'd always want to log to stderr because among the list of things we might want to log is "can't open syslog". Also, stderr is the right place to give the user an immediate error report on startup failures like "/var/run/quagga has wrong permissions".
- whether to log to syslog or not is a matter of user configuration. Unlike stderr, both syslog and file logging create persistent output so the user can't easily undo the effects of it. Syslog isn't the end-all be-all universal logging solution. cf. daemontools.
- a reasonable init system - e.g. systemd - will log error output from startup scripts into syslog (or some other log, e.g. /var/log/rc.log) anyway.
- if we log to stderr and syslog, we'll have duplicated logging.
- syslog is currently a singleton, but this is not neccessarily guaranteed to stay this way. I have pending work to add structured-data syslog (RFC5424) support, which ends up creating 2 different syslog targets. This work also has journald logging support, so there'd be 3 different system-log targets. On the other hand, there's only one stderr.